### ISSUE TITLE
BUG - Build/Compile options disabled for active project

### ISSUE THREAD

**Issue by bigangrydog** [eranif/codelite#1164]
I have a workspace with two projects. When I activate the second one, the build options in the main menu are disabled. I can, however, build the project by right-clicking on it and selecting the build context menu. Codelite version 9.1.2

Screenie attached.

![bug](https://cloud.githubusercontent.com/assets/17156238/13266005/d9feeb84-da6d-11e5-9724-5a385b71222a.png)

**Comment by eranif** [comment]
Out of curiosity, how did you manage to set a configuration name `Release 64` with space in the name?

**Comment by bigangrydog** [comment]
Like this ...

![bug](https://cloud.githubusercontent.com/assets/17156238/13266359/a8218566-da6f-11e5-8e4a-62befac868d3.png)

**Comment by bigangrydog** [comment]
I also notice that there's no way to get to the Configuration Manager from my active second project. I must activate the first one in order re-enable the menu options.

**Comment by bigangrydog** [comment]
Now I appear to have TWO active projects ...

![bug](https://cloud.githubusercontent.com/assets/17156238/13266442/08295a4c-da70-11e5-8ee5-85618c1173d0.png)

**Comment by eranif** [comment]
It should not allow this... oh well..
From the code, the only reason that the build option is disabled is this:
`bool enable = !ManagerST::Get()->IsBuildInProgress() && !ManagerST::Get()->GetActiveProjectName().IsEmpty();`

So: do you happen to have any unfinished build process of `mingw32-make` or similar?
and, can you upload the `.workspace` file when the problematic project is active?

last thing: if you close and open CodeLIte , does it change anything?

**Comment by eranif** [comment]
This looks like a corruption of an XML somewhere... this is why I asked for the `.workspace` file (having the `selftest.project` will also help

**Comment by bigangrydog** [comment]
>last thing: if you close and open CodeLIte , does it change anything?

Nope. Makes no difference. Re-running and opening the workspace (without) building, then activating the second project - the result is the same. Menus greyed out.

>This looks like a corruption of an XML somewhere... this is why I asked for the .workspace file

Yup. That's it. I edited it because it's the only way (I've found) to change the workspace name. I also changed a project name at the same time but missed changing it in the both the workspace and project files.
